manifest.toml

```diff
@@ -47,6 +47,11 @@ ram.runtime = "200M"
     # note: permissions are set in the install/upgrade/restore script
 
     [resources.data_dir]
+    dir = "/home/yunohost.app/__APP__"
```

Comment:
beware, i already set the correct permissions in the install/upgrade/restore scripts